Intersect.Client/Interface/Game/DescriptionWindows/DescriptionWindowBase.cs

```diff
@@ -167,5 +167,28 @@ public override void SetPosition(int x, int y)
 
             mContainer.MoveTo(newX, newY);
         }
+
+        public override void SetPosition(ImagePanel _icon, SpellDescriptionWindow _descriptionWindow)
```

Comment:
meaningful name, same name of rest above

Reply:
see https://github.com/AscensionGameDev/Intersect-Engine/pull/2277#discussion_r1641594174